ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -512,6 +804,110 @@ const triggerSet: TriggerSet<Data> = {
       netRegexJa: NetRegexes.startsUsing({ id: '6A2D', source: 'ヘスペロス', capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'P4S Act',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['6A0C', '6EB[4-7]', '6A36'], source: 'Hesperos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: ['6A0C', '6EB[4-7]', '6A36'], source: 'Hesperos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: ['6A0C', '6EB[4-7]', '6A36'], source: 'Hespéros' }),
+      netRegexJa: NetRegexes.startsUsing({ id: ['6A0C', '6EB[4-7]', '6A36'], source: 'ヘスペロス' }),
+      run: (data, matches) => {
+        const actMap: { [id: string]: string } = {
+          '6A0C': '1',
+          '6EB4': '2',
+          '6EB5': '3',
+          '6EB6': '4',
+          '6EB7': 'finale',
+          '6A36': 'curtain',
+        };
+        data.act = actMap[matches.id];
+      },
+    },
+    {
+      id: 'P4S Thorns Collector',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A0C', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6A0C', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6A0C', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6A0C', source: 'ヘスペロス', capture: false }),
+      promise: async (data, _matches, _output) => {
+        // Collect all Hesperos entities up front
```

Comment:
Should this just be done by `matches.source` instead of having a separate block of names?